kolibri/utils/server.py

```diff
@@ -151,7 +151,10 @@ def run_server(port):
     # Configure the server
     server.socket_host = LISTEN_ADDRESS
     server.socket_port = port
-    server.thread_pool = 30
+    server.thread_pool = conf.OPTIONS["Server"]["CHERRYPY_THREAD_POOL"]
+    server.socket_timeout = conf.OPTIONS["Server"]["CHERRYPY_SOCKET_TIMEOUT"]
+    server.accepted_queue_size = conf.OPTIONS["Server"]["CHERRYPY_QUEUE_SIZE"]
+    server.accepted_queue_timeout = conf.OPTIONS["Server"]["CHERRYPY_QUEUE_TIMEOUT"]
```

Comment:
for posterity, I was curious what the defaults of the other values were. Looks like:

| conf | val |
|--|--|
| `cherrypy.server.socket_timeout` | `10` |
| `cherrypy.server.accepted_queue_size` | `-1` |
| `cherrypy.server.accepted_queue_timeout` | `10` |

Reply:
@indirectlylit  yes, those are the defaults. You can check them in https://github.com/cherrypy/cherrypy/blob/3eb859974d4a5582fefb978916d1b60c7d3adc27/cherrypy/_cpserver.py#L57-L85